### PR TITLE
docs: Add banner search docs

### PR DESCRIPTION
**Pull request by igorbrasileiro**
## What's the purpose of this pull request?
This PR adds documentation about how to create a banner search for terms.

### CODE REVIEW

**Review by Mariana-Caetano — APPROVED** [review]
Awesome, @igorbrasileiro!

File: docs/how-to-create-banner-search.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Learn how to implement the Search Banner in the following sections.
+
+## Implementation Steps
+
+### Adding the Banner query
+
+To add the banner query you will need to shadow the `@vtex/gatsby-theme-store/templates/search.browser.tsx` file:
+
+1. Inside the source folder of your store project, create a file with this path `src/@vtex/gatsby-theme-store/templates/search.browser.tsx`
+2. After, add the following snippet of query on your search page query:
```
Comment (lariciamota) [inline docs/how-to-create-banner-search.md:16]:
```suggestion
2. Then, add the following query snippet on your search page query:
```

File: docs/how-to-create-banner-search.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Learn how to implement the Search Banner in the following sections.
+
+## Implementation Steps
+
+### Adding the Banner query
+
+To add the banner query you will need to shadow the `@vtex/gatsby-theme-store/templates/search.browser.tsx` file:
+
+1. Inside the source folder of your store project, create a file with this path `src/@vtex/gatsby-theme-store/templates/search.browser.tsx`
```
Comment (lariciamota) [inline docs/how-to-create-banner-search.md:15]:
```suggestion
1. Inside the source folder of your store project, open the file with this path `src/@vtex/gatsby-theme-store/templates/search.browser.tsx` or create it if it doesn't already exist.
```

File: docs/how-to-create-banner-search.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  }
+...
+`
+```
+
+> ℹ️ This query belongs to vtex.search-graphql app. Take a look at the [SearchPage](https://github.com/vtex-sites/storecomponents.store/blob/33904e9ed808c5496265da37206cfdaa9b0255d2/src/%40vtex/gatsby-theme-store/templates/search.browser.tsx#L142-L151) component.
+
+### Adding the Banner on UI
+
+You can access the banner data, on the AboveTheFold component.
```
Comment (lariciamota) [inline docs/how-to-create-banner-search.md:38]:
```suggestion
You can access the banner data on the `AboveTheFold` component. 
```

File: docs/how-to-create-banner-search.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+You can access the banner data, on the AboveTheFold component. 
+
+```tsx
+// src/@vtex/gatsby-theme-store/views/search/AboveTheFold.tsx
+
+const banners = data.vtex?.banners?.banners // List with all banners
+```
+
+So you can render all these banners HTML with this snippet.
```
Comment (lariciamota) [inline docs/how-to-create-banner-search.md:46]:
```suggestion
And you can render the HTMLs of all these banners with this snippet:
```

**Review by lariciamota — COMMENTED** [review]
🚀 🚀 
Just left some comments with possible improvements

File: docs/how-to-create-banner-search.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+)
+```
+
+> ℹ️ Take a look at the [AboveTheFold](https://github.com/vtex-sites/storecomponents.store/blob/7f8692706bc3b5a58df17d720228c55ce58de092/src/%40vtex/gatsby-theme-store/views/search/AboveTheFold.tsx#L33) and [SearchBanner](https://github.com/vtex-sites/storecomponents.store/blob/7f8692706bc3b5a58df17d720228c55ce58de092/src/components/search/Banner.tsx#L22-L25) components.
+
+### References
+
+- [Banner for search terms](https://github.com/vtex-sites/storecomponents.store/pull/1006).
+
+This [PR](https://github.com/vtex-sites/storecomponents.store/pull/1006) implements this feature on the Storecomponents store.
```
[the remark this answers is not in the document]
Comment (Mariana-Caetano) [reply docs/how-to-create-banner-search.md:62]: Great point, @lariciamota! 
What do you think we rewrite it taking some of the "this"?

` The PR, [Banner for search terms](https://github.com/vtex-sites/storecomponents.store/pull/1006), implements this feature on the Storecomponents store.`

[the remark this answers is not in the document]
Comment (igorbrasileiro) [reply docs/how-to-create-banner-search.md:62]: Sounds good @Mariana-Caetano

**Pull request merged**